armi/reactor/reactors.py

```diff
@@ -2303,6 +2303,12 @@ def processLoading(self, cs, dbLoad: bool = False):
         self.getNuclideCategories()
 
         # Generate list of flags that are to be stationary during assembly shuffling
+
+        if len(cs["stationaryBlocks"]) >= 1:
+            self.ValueError(
+                "stationaryBlocks setting is deprecated. Use stationaryBlocksFlags."
+            )
+
```

Comment:
Sorry, why is this here?  It looks like you are adding this setting back into ARMI, but telling people not to use i?

Reply:
Also probably a mistake when merging. It's removed now.